### ISSUE TITLE
Select name field when opening Filter

### ISSUE THREAD

**Issue by C9Glax** [C9Glax/tranga-website#79]
![image](https://github.com/C9Glax/tranga-website/assets/13404778/96692423-6eff-4289-8a49-8b50405c0d7d)
Would be great if it automatically focused this textbox when opening the filter.

**Comment by db-2001** [comment]
Yeah, I'm aware that this happens, it's definitely on the bugs list.

**Comment by C9Glax** [comment]
#159
